_posts/2022-04-11-1.15-announcement.md

```diff
@@ -0,0 +1,402 @@
+returning an error, which makes it more compliant with the SQL standard. The old 
+casting behavior can still be used by calling the newly introduced TRY_CAST function
+or restored via a configuration flag.
+
+In addition, many bugs have been fixed and improvements made to the casting 
+functionality, to ensure correct results. 
+
+
+### JSON functions
+
+JSON is one of the most popular data formats and SQL users increasingly need to build 
+and read these data structures.  Multiple 
+[JSON](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/functions/systemfunctions/#json-functions)
+functions have been added to Flink SQL 
+according to the SQL 2016 standard. It allows users to inspect, create, and modify JSON 
+strings using the Flink SQL dialect.
+
+
+## Community enablement
+
+Enabling people to build streaming data pipelines to solve their use cases is our goal. 
+The community is well aware that a 
+technology like Apache Flink is never used on its own and will always be part of a 
+bigger architecture. Thus, it is important that Flink operates well in the cloud, 
+connects seamlessly to other systems, and continues to support programming languages like Java and Python. 
+
+
+### Cloud interoperability
+
+There are users operating Flink deployments in cloud infrastructures from various 
+cloud providers. There are also services that offer to manage Flink deployments for 
+users on their platform. 
+
+In Flink 1.15, a recoverable writer for Google Cloud Storage has 
+been added. We also organized the connectors in the Flink ecosystem and put some focus 
+on connectors for the AWS ecosystem (i.e. 
+[KDS](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/kinesis/), 
+[Firehose](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/firehose/)).
+
+
+### The Elasticsearch sink
+
+There was significant work on Flink’s overall connector ecosystem, but we want to highlight the 
+[Elasticsearch sink](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/elasticsearch/) because it was implemented with 
+the new connector interfaces, which offers asynchronous functionality coupled with end-to-end semantics. 
+This sink will act as a template to follow in the future.
+
+
+### A Scala-free Flink
+
+A detailed 
+[blog post](https://flink.apache.org/2022/02/22/scala-free.html)  
+already explains the ins and outs of why Scala users can now use the Flink 
+Java API with any Scala version (including Scala 3). 
+
+In the end, removing Scala is just part of a larger effort of cleaning up and updating
+various technologies from the Flink ecosystem. 
+
+Starting in Flink 1.14, we removed the Mesos integration, isolated Akka, deprecated the
+DataSet Java API, and hid the Table API behind an abstraction. There’s already a lot of traction in the community towards these endeavors. 
+
```

Comment:
Could we add a section about PyFlink? It could be looked as following:
```
## PyFlink

Before Flink 1.15, Python user-defined functions will be executed in separate Python processes which will bring additional serialization/deserialization overhead and also communication overhead. In scenarios where the data size is big, e.g. image processing, etc, this overhead becomes non-negligible. Besides, since it involves inter-process communication, the processing latency is also non-negligible, which is unacceptable in scenarios where the latency is critical, e.g. quantitative transaction, etc. In Flink 1.15, we have introduced a new execution mode named 'thread' mode, where the Python user-defined functions will be executed in the JVM as a thread instead of a separate Python process. Benchmark has shown that the throughput could be increased by 2x in common scenarios such as json processing. The processing latency is also decreased from several seconds to micro-seconds. It should be noted that since this is still the first release of 'thread' mode, it currently only supports Python ScalarFunction which is used in Python Table API & SQL. We're planning to extend it to other places where Python user-defined functions could be used in the next releases.
```